taskcluster/ci/toolchain/qt.yml

```diff
@@ -44,6 +44,31 @@ qt-win:
         symbol: TL(qt-win)
     worker-type: b-win2012
 
+qt-win-next:
```

Comment:
Why is a new toolchain needed? Are we using a different Visual Studio on these images?

Reply:
We had a theory that we were locked out of updating our QT toolchain task due to the runner beeing on a too old windows. 
So ofc that's the first task i wanted to throw at it :D - turns out it that was the problem